admin-ui/plugins/auth-server/components/AuthN/LdapListingPage.js

```diff
@@ -0,0 +1,233 @@
+  const [selectedRow, setSelectedRow] = useState(null)
+  const toggle = useCallback(() => setModal(!modal), [modal])
+  const { permissions } = useSelector((state) => state.cedarPermissions)
+
+  SetTitle('LDAP Server')
+
+  function LdapDetailRow({ label, value }) {
+    return (
+      <Row>
+        <Col sm={6}>
+          <GluuFormDetailRow label={label[0]} value={value[0]} />
+        </Col>
+        {label[1] && value[1] && (
+          <Col sm={6}>
+            <GluuFormDetailRow label={label[1]} value={value[1]} />
+          </Col>
+        )}
+      </Row>
+    )
+  }
+
+  useEffect(() => {
+    const initPermissions = async () => {
+      const permissions = [LDAP_READ, LDAP_WRITE, LDAP_DELETE]
+      for (const permission of permissions) {
+        await authorize([permission])
+      }
+    }
+    initPermissions()
+    dispatch(getLdapList())
+  }, [dispatch])
+
+  useEffect(() => {}, [permissions])
+
+  useEffect(() => {
+    const actions = []
+    if (hasCedarPermission(LDAP_WRITE)) {
+      actions.push((rowData) => ({
+        icon: 'edit',
+        iconProps: { id: 'editLdap' + rowData.configId },
+        tooltip: `${t('actions.edit')}`,
+        onClick: () => {
+          dispatch(setCurrentItem({ item: rowData }))
+          navigate('/auth-server/authn/ldap/new', {
+            state: { isEdit: true },
+          })
+        },
+        disabled: !hasCedarPermission(LDAP_WRITE),
+      }))
+      actions.push({
+        icon: 'add',
+        tooltip: `${t('tooltips.add_ldap')}`,
+        iconProps: { color: 'primary', style: { color: customColors.lightBlue } },
+        isFreeAction: true,
+        onClick: () => {
+          navigate('/auth-server/authn/ldap/new', {
+            state: { isEdit: false },
+          })
+        },
+        disabled: !hasCedarPermission(LDAP_WRITE),
+      })
+    }
+    if (hasCedarPermission(LDAP_DELETE)) {
+      actions.push((rowData) => ({
+        icon: 'delete_outline',
+        iconProps: {
+          color: 'primary',
+          style: {
+            color: customColors.darkGray,
+          },
+          id: 'deleteLdap' + rowData.configId,
+        },
+        tooltip: `${t('actions.delete')}`,
+        onClick: (_event, rowData) => {
+          setSelectedRow(rowData)
+          setModal(true)
+        },
+        disabled: !hasCedarPermission(LDAP_DELETE),
+      }))
+    }
+    setMyActions(actions)
+  }, [hasCedarPermission, t, dispatch, navigate])
+
+  useEffect(() => {
+    if (ldapListRaw && ldapListRaw.length > 0 && !loading) {
+      const processed = ldapListRaw.map((item) => ({
+        ...item,
+        configId: item.configId,
+        bindDN: item.bindDN,
+        enabled: item.enabled,
+      }))
+      setLdapList(processed)
+    } else {
+      setLdapList([])
+    }
+  }, [ldapListRaw, loading])
+
+  return (
+    <>
+      <GluuViewWrapper canShow={hasCedarPermission(LDAP_READ)}>
+        <MaterialTable
+          key={limit ? limit : 0}
```

Comment:
Drop constant key to prevent unnecessary remounts.

Using a static key forces MaterialTable to remount on prop changes.

```diff
-          key={limit ? limit : 0}
```